contribution/src/main/java/de/promptheus/contributions/service/MeilisearchService.java

```diff
@@ -349,6 +351,9 @@ public String hybridSearch(String query, String user, String week, int limit) {
             } else {
                 log.info("Performing text-only search (vector embedding failed)");
             }
+            */
+
+            searchParams.put("hybrid", Map.of("semanticRatio", 0.5)); // 50% semantic, 50% keyword
```

Comment:
Searching should be done in the search service, but yes, this is missing there

Reply:
Maybe later